src/components/App.js

```diff
@@ -1,14 +1,38 @@
+import React from 'react';
+// import ReactDOM from 'react-dom';
```

Comment:
Remove this and any other comments from the codebase. Comments make your code look ugly.